npm-packages/meteor-rspack/lib/localDependenciesHelpers.js

```
@@ -0,0 +1,180 @@
+
+      for (const ext of extensions) {
+        const pathWithExt = resolvedPath + ext;
+        if (fs.existsSync(pathWithExt)) {
+          resolvedPath = pathWithExt;
+          found = true;
+          break;
+        }
+      }
+
+      // If not found with extension, try index files in directory
+      if (!found && fs.existsSync(resolvedPath) && fs.statSync(resolvedPath).isDirectory()) {
```

ESLint

[error] Replace `!found·&&·fs.existsSync(resolvedPath)·&&·fs.statSync(resolvedPath).isDirectory()` with `⏎········!found·&&⏎········fs.existsSync(resolvedPath)·&&⏎········fs.statSync(resolvedPath).isDirectory()⏎······`

(prettier/prettier)